src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -514,7 +522,7 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
   }
 
   /** Should fail if a value is already set.  The keys in the Map are locally qualified names. */
-  override def setOutputs(workflowId: WorkflowId, key: OutputKey, callOutputs: WorkflowOutputs): Future[Unit] = {
+  override def setOutputs(workflowId: WorkflowId, key: OutputKey, callOutputs: WorkflowOutputs, topLevelOutputs: Map[FullyQualifiedName, WdlType]): Future[Unit] = {
```

Comment:
Hey now, "the Jeff way" is how you're supposed to functional scala :)

For my own purposes I was thinking about making this a real type via case class so I'd support it